### ISSUE TITLE
v0.9.3 Chat is unusable

### ISSUE THREAD

**Issue by jbob182** [NodeBB/NodeBB#3980]
We rely on the chat feature very heavily, but when we upgraded to v0.9.3 it was so unstable we were forced to disable it. (https://forums.fogproject.org/topic/6374/chat-is-disabled). Here are the main issues our users noticed before we disabled it.

1.  We are unable to chat individually to users, with the warning "This user has restricted their chat messages. They must follow you before you can chat with them". Our forums rely on any user being able to message another. In addition, we were unable to find any setting in the admin console to change the default behavior or disable this functionality.

2. If you have a group chat, and also a chat room with only 1 user, the two would "merge", adding all members of the group chat to the individual chat.  The "merging" would happen if you went to the chat page and selected the group chat and then the individual chat. 

3. Notifications. Even if you have focus on the chat window and a user replies in that chat, you will still receive a notification a few minutes later.

I apologize if any of the issues are incorrect, and no doubt you will need to verify them.  This is just a list of what issues we and our users saw before we disabled it, some issues may straight out wrong or even partially correct. We'll do our best to try and verify these issues.

**Comment by jbob182** [comment]
Update on #2.  We have confirmed users being added to chats. If you open a second chat pop-up window the user gets added to the first window. See attached gif.
![chat-2](https://cloud.githubusercontent.com/assets/1146681/11920620/57576af0-a73a-11e5-8f4d-78961e3d60a8.gif)

**Comment by barisusakli** [comment]
Weird, group chat isnt even in 0.9.3 its only on master. Can you post `git rev-parse HEAD` and `npm ls nodebb-theme-persona`

**Comment by pichalite** [comment]
@barisusakli I think people are getting confused coz master displays NodeBB version as 0.9.3 in the ACP.

**Comment by jbob182** [comment]
@barisusakli as usual, its the user's (our) fault.  Somehow our upgrade put us on the latest master branch commit  when we pulled. (We were on 80fd0b20bdd07a539734d8b9f56429197ef7c18e) 

I've looked over our bash history file and when the upgrade took place, the checkout command was only partially correct, making a branch `v0.9.3` but not linking it to the tag. Sorry about any confusion. I would agree with @pichalite because one of the first things we did was look at out ACP to verify we were on v0.9.3.
